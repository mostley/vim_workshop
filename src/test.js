const fs = require("fs");

describe("Exercises", () => {
  describe.each([["username1"], ["username2"]])("%s", (username) => {
    it.each([["x.txt"]])("%s", (filename) => {
      const data = fs.readFileSync(`./src/${username}/${filename}`, {
        encoding: "utf8",
        flag: "r",
      });
      expect(data).toMatchSnapshot();
    });
  });
});
