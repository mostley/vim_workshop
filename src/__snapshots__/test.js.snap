// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exercises username1 x.txt 1`] = `
"________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
"
`;

exports[`Exercises username2 x.txt 1`] = `
"________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
________________________________________
"
`;
